core/modules/system/system.install

```diff
@@ -3838,6 +3838,27 @@ function system_update_1103() {
   $config->save();
 }
 
+/**
+ * Update existing Views with grid display to be unresponsive.
+ */
+function system_update_1104() {
```

Comment:
Shouldn't that actually go to file views.install?

Comparable to views_update_1005()...